builtin/providers/aws/resource_aws_flow_log_test.go

```diff
@@ -0,0 +1,209 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/aws-sdk-go/aws"
```

Comment:
Wow, great catch! My editor must have auto populated that :scream: 

Fixed in 0bf127a